sale_layout_category_hide_detail/tests/test_sale_layout_category_hide_detail.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.addons.sale.tests.common import SaleCommon
+
+
+class TestSaleLayoutCategoryHideDetail(SaleCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
```suggestion
```
It is no longer necessary, SaleCommon inherits from ProductCommon, which in turn inherits from BaseCommon.